### PR TITLE
Cleanup: Fix small warnings

### PR DESCRIPTION
**Pull request by nonZero**

### CODE REVIEW

File: client/src/components/Mapa.jsx [hunk 1]
```diff
@@ -55,8 +55,8 @@ const Mapa = (props) =>  {
 				geom && <GeoJSON key={geomHash} data={geom} />
 			}
 			<div className="map-title">
-				{countyName && <button className="btn btn-light disabled">{countyName}</button>}
```
Comment (florpor) [inline client/src/components/Mapa.jsx:58]: this changes the way the texts in these "buttons" are styled which needs to be fixed.
plus placeholder buttons were not converted (lines 14-15)

File: client/src/scenes/TreePermits.jsx [hunk 2]
```diff
@@ -124,15 +124,14 @@ class TreePermits extends Component {
 							this
 						)}
 					/>
-					<Typography variant="paragraphText" mobileVariant="paragraphText"
```
Comment (florpor) [inline client/src/scenes/TreePermits.jsx:127]: this changes the typography font.
can't say that i understand why it worked until now, but along with deleting the variants font-family should be set to Assistant

**Review by florpor — CHANGES_REQUESTED** [review]
hey @nonZero , i finally got to start reviewing your PRs.
a couple changes here caused style changes which we won't want to deploy.
will you be able to fix those changes?

File: client/src/components/Mapa.jsx (hunk 1 above)
Comment (nonZero) [reply client/src/components/Mapa.jsx:58]: hmmm... I am not sure I understand your comment.
Is there any change in rendering between `<button>` and `<span>`?
I believe the style comes from `className="btn btn-light..."`
I think Lines 14-15 do not appear inside another `<button>` so they do not have issues.

File: client/src/scenes/TreePermits.jsx (hunk 2 above)
Comment (nonZero) [reply client/src/scenes/TreePermits.jsx:127]: Hmm.... I don't think this code change has any effect on rendering.
There are two `Typography` components in this project - `"@material-ui/core/Typography"` and a custom one in meireim's code), is there a mixup here?

**Pull request merged**